scripts/verify_adv_IM.py

```diff
@@ -0,0 +1,113 @@
+# script to quickly check the status of adv_im for a event
+# checks for existance of aggregated csv
+# checks for nan/null in the csv
+# checks station in csv matches stations ran
+
+
+import argparse
```

Comment:
Nitpicking, but can thoughts on changing the order of the imports as per, https://www.python.org/dev/peps/pep-0008/#imports?
I'm def a fan of this one, and think it would be a good idea to this across the team.

Also use docstring notation for the script docstring above? (https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)

Reply:
I agree. i've re-ordered them and change the comments to docstring.